### ISSUE TITLE
Create a user guide for content editors

### ISSUE THREAD

**Issue by verythorough** [netlify/netlify-cms#1340]
This is pulled from the docs re-org issue https://github.com/netlify/netlify-cms/issues/598#issuecomment-332564210

The current docs focus mostly on "implementers" (people writing code to configure and customize Netlify CMS on their site) and a little bit on "contributors" (people making contributions to the CMS itself). So far, they don't provide any guidance for content editors using the CMS UI to add and edit content.

I'm thinking that initially, we can put the whole guide in a single page of the docs, but I suspect we'll want to break it out into a multi-page section when we've decided how we want that to work. For now, I think getting any guidance for editors would be helpful.

Content-wise, I'm thinking this is not a place to explain what’s happening “under the hood,” though we may include links here and there for the curious. Here's a rough outline brainstorm to get anyone started who might like to take this on:

- Accessing the CMS UI: may vary, but describe common use case of website.com/admin
  - troublehooting access?
- UI overview
  - Sidebar
    - editorial workflow - brief description, link to dedicated section
    - collections
      - show existing content
      - create new content
- Content editor
  - Accessed by selecting an existing item or creating a new one
  - Brief explanation of fields and preview
    - required/optional
    - not all field values displayed in preview
    - special widgets like image (link to media library section)
  - Rich text editor
    - Markdown mode
    - Inserting blocks (like image)
  - What “Save” does (depending on whether Editorial Workflow is enabled)
- Media library
  - adding/deleting/searching for assets
  - how to add to content (link to content editor)
- Editorial Workflow
  - getting content into the workflow (by saving changes)
  - moving centent between columns and what that means
  - publishing from the ‘Waiting to go live’ column

**Comment by JRPfeiffer1** [comment]
I came up with a couple of paragraphs that might be useful as an introduction to the content editor’s user guide:

Netlify CMS is a fast, safe, and scalable content management system that comes with rich-text editing, real-time previews, and drag-and-drop media uploads. Its intuitive workflow makes it easy to manage content from draft to review to publish—across any number of content types.

Because it’s open source, Netlify CMS is supported by a growing community of developers. They love the fact that it works seamlessly with Git—but thanks to the work they’ve done, you’ll probably never have to think about that.

**Comment by erquhart** [comment]
Thanks @JRPfeiffer1!

@verythorough thanks for putting this together, that outline looks like a solid start.

**Comment by tech4him1** [comment]
I think it would be a great idea to add "help links" from inside the CMS once this documentation is written.

**Comment by jorear81** [comment]
I would like to tackle Media Library.

**Comment by verythorough** [comment]
Great, @jorear81! I think we'll be doing some re-arranging (and changing the media library functionality a bit, too), but until then, this will still be a useful resource for others. 

How about we create a "Media Library UI" page in the Reference group of pages for now. We can add more sections and put them into a larger guide later.

**Comment by erquhart** [comment]
I don't believe there's any work on the content side of this, no. Open for contribution 👍

**Comment by tomrutgers** [comment]
Happy to provide feedback to ideas etc @ademuro

**Comment by ademuro** [comment]
@tomrutgers yes, please! that would be great. i'll start by spending time getting to know the CMS better and then write up a draft to share with you and @verythorough (if she's interested). would that work? i appreciate any feedback and i look forward to working on this :)

**Comment by tomrutgers** [comment]
Sure! If you have questions about the CMS join us on [Gitter](https://gitter.im/netlify/NetlifyCMS)

**Comment by pungggi** [comment]
I am at your disposal for any translation in german and italian..

**Comment by ademuro** [comment]
Besides Netlify Identity, I'm wondering what other identity services CMS users might use to access the CMS? For this documentation there is a section on how to access the CMS, but I am not sure how we want to address it since I believe there are a lot of different ways to access it?

**Comment by tomrutgers** [comment]
Apart from custom solutions, in terms of ways to log in there are two:
- With a user name / password (Only available with Netlify Identity)
- With external auth
  1 Github
  2 GitLab
  3 Bitbucket
  _4 Google OAuth (available as external provider when using Netlify Identity)_

**Comment by erquhart** [comment]
You said that so much better lol, thank you

**Comment by ademuro** [comment]
Thanks, Shawn and Tom! I'll post my draft in this comment thread today.

**Comment by verythorough** [comment]
Regarding #2, yoursite.com/admin/ is by far the most common route for logging in, but it's not guaranteed. 

The degree of possible variability in the login process makes documenting it difficult. It's impossible to document every conceivable possibility, so I see two alternatives:

1. Tell the user to contact their "site administrator" for login instructions. This will likely be part of the process for most users anyway. 
2. Document the single most common procedure, with a note that configurations may vary, and users should contact their site admin for details if the instructions don't match their experience. 

I think 1 is shippable, but 2 would be more complete. For the "single most common procedure," I would match what's in the quick start templates (and be sure they match each other).

**Comment by tomrutgers** [comment]
To be honest, the users that need guides like these probably won't sign in using GitLab anyway. Netlify Identity + Git Gateway (username + password) is the most common practice if I have to take a guess.

**Comment by dopry** [comment]
I think getting something shippable our and revising based on feedback as good.  It would be nice to have a document or anchor for each login flow.  As a site builder, I can link my clients to the correct content in our hand-off documents. I would prefer pages for each flow that varies like that so I don't link a user to an in-page anchor and then they scroll elsewhere and get confused.

I only think this is really important for login as it's a user's first impression. Where the optionality can be explained in the documentation such as Workflow and editing I think it is fine to keep inline.  It might make make sense to break our Media handling by the back-end. Cloudinary is definitely very different at least visually than managing media in Git. 

@ademuro's document looks like a great start.

**Comment by cassandraoid** [comment]
Are there any docs for content editors I am missing (I see this is an old issue)? Going to be using the project for a new site and want to be able to reference my editors to something :) looked through the docs but seems to be targeted toward the devs.

**Comment by erquhart** [comment]
Hey Cassandra - you're not missing anything, this just hasn't happened yet. I'm going to take a quick run at docs improvements next week, I can try to cook up at least a guide if you can mention any specific areas that you're hoping to see user docs for.

**Comment by verythorough** [comment]
Hi @cassandraoid!! Until we do gets these into the proper docs, @ademuro's draft above is a start: https://paper.dropbox.com/doc/Draftbrainstorm-for-content-editor-guide-Netlify-CMS--AfCBZPzCD2vc54VzU0oVv7gqAg-KemvQi07ln8itRhaZ1Gbd
